src/prefect/orion/api/flow_runs.py

```diff
@@ -155,7 +156,7 @@ async def read_flow_run_graph(
     )
 
 
-@router.post("/filter")
+@router.post("/filter", response_class=ORJSONResponse)
```

Comment:
Will this break our typing in the API reference?

Can we introduce a custom response type that includes a generic e.g. `JSONResponse[List[FlowRun]]`? Or another more elegant solution?

Reply:
It does break the API reference documentation:
- https://deploy-preview-6553--prefect-orion.netlify.app/api-ref/rest-api/#/Flow%20Runs/read_flow_runs_flow_runs_filter_post 
- https://main--prefect-orion.netlify.app/api-ref/rest-api/#/Flow%20Runs/read_flow_runs_flow_runs_filter_post

(Yay deploy previews)

Reply:
Good catch! Adding back the type hint is fine as long as we return `ORJSONResponse` directly. Docs looked good for me locally.

Reply:
Nice